module-2/README.md

```diff
@@ -134,26 +117,23 @@ When done testing the service you can stop it by pressing CTRL-c on PC or Mac.
 
 #### Pushing the Docker Image to Amazon ECR
 
-With a successful test of our service locally, we're ready to create a container image repository in Amazon Elastic Container Registry (Amazon ECR) and push our image into it.  In order to create the registry, run the following command, this creates a new repository in the default AWS ECR registry created for your account. Run the below command using either the AWS CLI or the PowerShell command:
+With a successful test of our service locally, we're ready to create a container image repository in Amazon Elastic Container Registry (Amazon ECR) and push our image into it.  In order to create the registry, run the following command, this creates a new repository in the default AWS ECR registry created for your account. 
 

```

Comment:
Same, can we let people write this code? It should be quite short and straightforward with the CDK. That's the whole point of the CDK, right? So we don't need people to copy & paste solutions in order to use AWS, but rather leverage high level constructs.